File: lab-3-ddb/environment/ruby_2.6.0/edit_item.rb
require "aws-sdk-dynamodb"

dynamodb_client = Aws::DynamoDB::Client.new(region: "<FMI>")

table_name = "lostcats"
petname = "Hosepipe"

params = {
        table_name: table_name,
        key: {
                petname: <FMI>
        },
        update_expression: "set breed = :b",
        expression_attribute_values: {
                ":b" => "British Shorthair"
        },
        return_values: "<FMI>"
}

begin
        dynamodb_client.<FMI>(params)
    puts "Updated Item"
end

File: lab-3-ddb/environment/python_3.6.8/edit_item.py
import boto3

DDB_RESOURCE = boto3.resource("dynamodb", region_name="<FMI>")

table = DDB_RESOURCE.Table("lostcats")

petname = "Hosepipe"

table.<FMI>(
    Key={
        'petname': petname
    },
    UpdateExpression="set breed = :b",
    ExpressionAttributeValues={
        ":b": "British Shorthair"
    },
    ReturnValues="<FMI>"
)